### PR TITLE
Fixed bugs in codes used when not clustering

### PR DESCRIPTION
**Pull request by idorrington92**
Two trivial fixes

### CODE REVIEW

**Comment by spxiwh** [comment]
Hi @idorrington92, thanks for making these changes. I'm happy to approve them. However, I cannot merge these changes at present because the sanity tests are failing. They are failing because the branch is a long way behind master and some updates are needed for Travis. Can you *rebase* the branch against the current origin/master. If that doesn't make sense; pop into @duncanmmacleod's office!

File: pycbc/events/threshold_cpu.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def threshold_numpy(series, value):
     arr = series.data
     locs = numpy.where(arr.real**2 + arr.imag**2 > value**2)[0]
     vals = arr[locs]
-    return locs, vals
+    return vals, locs
```
Comment (ahnitz) [inline pycbc/events/threshold_cpu.py]: I think we need to be consistent on the output order. The other single function thresholding functions have the original order. The exception is the threshold and cluster function, which I agree is inconsistent overall, but I think this change makes it less so. Can you get away with just changing the order where you are calling this function?

Comment (spxiwh) [reply pycbc/events/threshold_cpu.py]: @ahnitz You mean swap order here https://github.com/idorrington92/pycbc/blob/26c68e2d0781dd76c7b3a73af4cbf034ccd1dc42/pycbc/filter/matchedfilter.py#L339 instead?

Comment (ahnitz) [reply pycbc/events/threshold_cpu.py]: Yes, that's what I mean.

**Comment by spxiwh** [comment]
@idorrington92 @ahnitz Can this get sorted at next week's face-to-face?

**Comment by ahnitz** [comment]
@idorrington92 @spxiwh What's the status of this?

**Comment by spxiwh** [comment]
Closing in favour of #2476

**Pull request closed without merging**